cluster/charts/rook-ceph/values.yaml

```diff
@@ -70,6 +70,86 @@ csi:
   # Allow starting unsupported ceph-csi image
   allowUnsupportedVersion: false
 
+   # (Optional) CEPH CSI RBD provisioner plugin resource requirement list, Put here list of resources
+  # requests and limits you want to apply for provisioner plugin container
+  # csiRBDProvisionerPluginResource:
```

Comment:
Can we implement it such that you can specify the container name and the resources would be applied to that container? Then we don't need individual names for all the resources. For example, the resources would be key-value pairs where the key is the container name.  This means we would only need four new settings, each of which is structure as a list keyed off the container names. Here is an example of the rbd provisioner:
```
csiRBDProvisionerResource:
- name: csi-rbdplugin
    requests:
      ...
    limits:
      ...
- name: csi-provisioner
     requests:
      ...
     limits:
      ...
- name: <container-name>
     requests:
      ...
     limits:
      ...
```